### PR TITLE
Correcting one of the usage in documentation

### PR DESCRIPTION
**Pull request by amalkurup89**
**What this PR does / why we need it**: Documentation fix

**Which issue(s) this PR fixes**: Correcting one of the usage in documentation

### CODE REVIEW

**Comment by amalkurup89** [comment]
@oddlittlebird Can you please review

**Comment by amalkurup89** [comment]
@aknuds1 Can you please review

File: docs/sources/variables/advanced-variable-format-options.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Interpolation result: 'test1,test2'
 
 ## Distributed - OpenTSDB
 
-Formats multi-value variable in custom format for OpenTSDB.
+Formats multi-valued variable in custom format for OpenTSDB.
```
Comment (oddlittlebird) [inline docs/sources/variables/advanced-variable-format-options.md:39]:
```suggestion
Formats variables with multiple values in custom format for OpenTSDB.
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
The original wording is confusing. Here's my suggestion, feel free to improve it if you think of something better.

**Comment by amalkurup89** [comment]
> The original wording is confusing. Here's my suggestion, feel free to improve it if you think of something better.

Looks good to me as well. Do you have any more suggestions

**Comment by amalkurup89** [comment]
@oddlittlebird Can you please merge, if changes look good to you

**Review by oddlittlebird — APPROVED** [review]
LGTM. Thanks for contributing!

**Pull request merged**